content/ja/docs/languages/sdk-configuration/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: SDKの設定
+linkTitle: SDKの設定
+aliases: [/docs/concepts/sdk-configuration]
+redirects: [{ from: /docs/concepts/sdk-configuration/*, to: ':splat' }]
```

Comment:
We don't actually want this redirects to be duplicated:

```suggestion
```

I'll create a followup issue for this, but please drop the line for now.

Reply:
Thank you for catching this. I remove this line.

Reply:
Will track this via:

- #6544